### PR TITLE
Add md5 extension

### PR DESCRIPTION
**Pull request by OscarRG**
## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by illyakurochkin — APPROVED** [review]
Great job!

File: extensions/md5/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "icon": "command-icon.png",
+  "author": "oscarrg",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "String 2 MD5 Hash",
```
Comment (pernielsentikaer) [inline extensions/md5/package.json:15]:
```suggestion
      "title": "Copy MD5 Hash",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Instead of using a form you could make it a `no-view` command and use an [argument](https://developers.raycast.com/information/lifecycle/arguments) to get the data instead

You can also combine it with [getSelectedText](https://developers.raycast.com/api-reference/environment#getselectedtext)

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by pernielsentikaer** [comment]
Is this ready for another review?

**Comment by erics118** [comment]
https://www.raycast.com/eth-p/encoding-tools has an `Encode MD5` command.

**Comment by pernielsentikaer** [comment]
Great catch, @erics118. There are many hidden gems in the store which do not always pop up (not sure why it didn't for me at first).

@OscarRG, could you check out the above extension? Sorry that I didn't catch it before we started the review.

**Pull request closed without merging**